runtimes/ocw-runtime/src/lib.rs

```diff
@@ -1,4 +1,4 @@
-//! This Runtime demonstrates how to config signed and unsigned transaction handler to be used
+//! This Runtime demonstrates how to configure signed and unsigned transaction handler to be used
```

Comment:
```suggestion
//! This runtime demonstrates how to configure signed and unsigned transaction handler to be used
```

Reply:
Should it be "transaction handlers" perhaps?

Reply:
```suggestion
//! This runtime demonstrates how to configure signed and unsigned transaction handlers to be used
```